src/sentry/static/sentry/app/views/alerts/rules/index.tsx

```diff
@@ -226,3 +222,40 @@ const StyledSortLink = styled(Link)`
     color: inherit;
   }
 `;
+
+const StyledPanelTable = styled(PanelTable)`
+  ${PanelTableHeader} {
+    line-height: normal;
+  }
+  grid-template-columns: auto 1.5fr 1fr 1fr 1fr auto;
```

Comment:
So I would get @robinrendle 's opinion on this but I would change the column templates so that the "Alert Name" column gets the most space. We can do this by making every column `max-content` and the alert name to be `auto`. This way, every `max-content` column will only ever be as wide as the widest element in that column, and the `auto` column will fill in the rest.

Reply:
Pushed all the changes and also spoke with Robin on the columns and he said using `grid-template-columns: auto 1.5fr 1fr 1fr 1fr auto;` is better so the Created By column isn't too squished. To use this, I do need to keep `white-space: nowrap;` (from above) because the line-height gets a little weird when you resize the window to smaller widths.